.lintstagedrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
we should move this into the client folder

Reply:
+1 @nwochaadim. However, if it's possible to have the config in package.json, that will be more succinct. Well done @williamolojede

Reply:
@nwochaadim @adesege the reason why I have it at the root directory is that the backend team might also need to set pre-commit hooks in the future. Is that assumption valid or I should go ahead to move it into the client folder

Reply:
Thanks for the explanation. However, leaving this at the root is a team-wide decision. Does moving it to package.json work as @adesege suggested? If yes, that might be better

Reply:
yeah, it will work. My reason for not adding it to package.json is so that it's not doing plenty of things.